spec.bs

```diff
@@ -2191,6 +2279,26 @@ Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 # Structures # {#structures}
 
+<xmp class="idl">
+
+dictionary PreviousWin {
+  required long long timeDelta;
+  required DOMString adJSON;
+};
+
+dictionary BiddingBrowserSignals {
+  required DOMString topWindowHostname;
+  required USVString seller;
+  required double joinCount;
+  required double bidCount;
+
+  USVString topLevelSeller;
+  sequence<PreviousWin> prevWinsMs;
+  object? wasmHelper;

```

Comment:
Is there a different meaning between null and undefined/absent for this field? If not:

```suggestion
  object wasmHelper;
```